e2e/ui/global-setup.js

```diff
@@ -0,0 +1,24 @@
+const { chromium } = require('@playwright/test');
+
+module.exports = async config => {
+
+  var NOMAD_TOKEN = process.env.NOMAD_TOKEN;
+  if (NOMAD_TOKEN == undefined || NOMAD_TOKEN == "") {
+    return
+  }
```

Comment:
This could be done in a followup PR, but would it make sense to allow running the tests in an environment without ACLs? Like against a local dev agent.

Reply:
This global setup already handles that case fine because it returns early if no `NOMAD_TOKEN` is set. But we'd need to make any tests that checks/requires auth to return early as well. I'll leave that up to test authors for follow-up PRs.